### ISSUE TITLE
Migrate scan job to new job services

### ISSUE THREAD

**Issue by steven-zou** [vmware/harbor#4331]

**Comment by reasonerjt** [comment]
Pending on integration with web hooks.

**Comment by reasonerjt** [comment]
The code has been merged into `job_service` branch, close this issue.  
Further bugfix or cosmetic changes will be tracked by new issues.
